.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: cpp
 
-dist: trusty
+#dist: trusty
```

Comment:
I see that the original file had a lot of commented out lines.  They are removed now.  Can you please pull and rebase against it again?